snorkel/mtl/loggers/log_manager.py

```diff
@@ -0,0 +1,174 @@
+import logging
+import os
+from datetime import datetime
+from typing import Dict
+
+import torch
+
+from snorkel.mtl.loggers import logging_default_config
```

Comment:
e.g. `from .checkpointer import ...` is fine here

Reply:
PEP8 recommends absolute imports ("Absolute imports are recommended, as they are usually more readable and tend to be better behaved (or at least give better error messages) if the import system is incorrectly configured"). Will change here for now though to be consistent with the files you've already moved over. Can discuss and update later if need be.